doc/source/changes/version_0_28.rst.inc

```diff
@@ -234,6 +233,50 @@ Miscellaneous improvements
 
   Closes :issue:`548`:
 
+* replaced argument `transpose` by `wide` in `to_csv` and `to_excel` methods. The new argument `wide` has
```

Comment:
maybe split this in two separate sections: 
* to_csv(transpose -> wide), which does not need any example/explanation because it is a simple rename.
* to_excel(transpose -> wide) which does need an explanation since the behaviour is slightly different